frontend/src/modules/editor/global/templates/partials/part_editorCommon.hbs

```diff
@@ -33,39 +33,39 @@
   <div class="sidebar-row">
     <button class="editor-common-sidebar-project">
       <span class="editor-common-sidebar-project-inner">
-        <i class="fa fa-chevron-right primary-color"/>{{ t 'app.projectsettings'}}
+        <i class="fa fa-chevron-right primary-color"/>{{ t 'app.editorsettings'}}
       </span>
     </button>
   </div>
 
   <div class="sidebar-row">
     <button class="editor-common-sidebar-config">
       <span class="editor-common-sidebar-config-inner">
-        <i class="fa fa-chevron-right primary-color"/>{{ t 'app.configurationsettings'}}
+        <i class="fa fa-chevron-right primary-color"/>{{ t 'app.editorconfig'}}
       </span>
     </button>
   </div>
 
   <div class="sidebar-row">
     <button class="editor-common-sidebar-select-theme">
       <span class="editor-common-sidebar-select-theme-inner">
-        <i class="fa fa-chevron-right primary-color"/></i>{{ t 'app.themepicker'}}
+        <i class="fa fa-chevron-right primary-color"/></i>{{ t 'app.editorselecttheme'}}
```

Comment:
String not present in `en-application.json`.

Should be left as `app.themepicker`?